net/dpdk/Makefile

```diff
@@ -0,0 +1,38 @@
+# SPDX-License-Identifier: GPL-2.0-only
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME := dpdk
+PKG_VERSION := 21.05
+
+PKG_SOURCE := dpdk-$(PKG_VERSION).tar.xz
+PKG_SOURCE_URL := http://static.dpdk.org/rel
+PKG_HASH := 1e12499b4c5fcdb57c83e5fe184ea6becddf7cf092893c2c5ef2efb0eec12f0b
+
+PKG_BUILD_DEPENDS:= meson/host
+include $(INCLUDE_DIR)/package.mk
+include $(INCLUDE_DIR)/host-build.mk
+include $(TOPDIR)/feeds/packages/devel/meson/meson.mk
+
+TARGET_CFLAGS += -D_GNU_SOURCE -Wno-unused-result -Wno-format-nonliteral
+
+define Package/dpdk
+  SECTION    := net
+  CATEGORY   := Network
+  TITLE      := Data Plane Development Kit (DPDK)
+  DEPENDS    := @(x86_64||arm||aarch64) +librt +libpthread +libbpf +libpcap +libopenssl
```

Comment:
Doesn't DPDK also work for x86 (32-bit)?

Reply:
Yes @pprindeville ..DPDK do support 32 bit version. But we have not tested yet.